bencyn/Questioner-React
# #view meetup details

#### What does this PR do?

This PR implements meetup detail view

### Tasks to be completed?
- [x] - implement view meetup component
- [x] - fetch a meetup and display on the homepage

### How can this be manually tested?

- Clone the repository  
  `git clone https://github.com/bencyn/Questioner-React.git`

-  Check out to current branch
    `git checkout view-meetup`

- setup npm
`$ npm install`

- run application
`$ npm run dev`
